### PR TITLE
Update Changelog with last changes and Readme with more details

### PR DESCRIPTION
**Pull request by Trim**
Add details to Readme about our community driven project.

Update changelog with recent improvements.

Note: this PR comes from a split of #70

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,102 @@
...
+Exchange Calendar is an add-on for [Thunderbird](https://mozilla.org/thunderbird)
+and [Seamonkey](https://www.seamonkey-project.org/).
+
+It extends the [Lightning](https://mozilla.org/calendar) add-on to provide
+[Microsoft Exchange](http://microsoft.com/exchange) support.
+
+Exchange Calendar extension can:
+  * Synchronize calendars, tasks, and contacts with a Microsoft Exchange server.
+  * Create, edit and update events and tasks.
+  * Display contacts from your address books and the global address list
```
Comment (advancingu) [inline README.md]: It is recommended that all bullet points that form complete sentences end with a period (full stop), so please add those here and below.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,102 @@
...
+The ExchangeCalendar community is now maintaining this extension. Support is
+best effort and cannot be guaranteed. Contributions are welcome.
 
 ## Contributing
 
 This add-on is open source and based on the work of many
 [contributors](https://github.com/ExchangeCalendar/exchangecalendar/graphs/contributors).
 
-### Building
+This project is a community driven effort to keep maintained and uptodate
```
Comment (advancingu) [inline README.md]: Suggestion:
> This project is a community driven effort to develop and maintain the "Exchange EWS Provider" extension originally created by Michel Verbraak.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,102 @@
...
+the "Exchange EWS Provider" extension created by Michel Verbraak.
+
+Currently, the community is really small and have too few developers to
```
Comment (advancingu) [inline README.md]: Any help to move this project forward is always appreciated.

I would move the warranty sentence into a separate heading "Warrany" before "License" and state:

> This software is provided on a best-effort basis without any warranty. Please see the [license](LICENSE) file for details.

File: README.md [hunk 4]
```diff
@@ -1,19 +1,102 @@
...
+Please note that the project is built only for latest stable Thunderbird release
+and latest stable Lightning release.
+
+If you use any different release, please write it in your report.
+
+As this add-on extends Thunderbird and Lightning directly, it could be in
+conflict with other add-ons.
 
-To compile the add-on, run the `build.sh` script.
+If you can give us a list of installed and enabled add-ons, it will help us.
```
Comment (advancingu) [inline README.md]: Suggestion:

> The more information you can provide about an issue, the easier it becomes to reproduce and fix. Examples of helpful information are:
    * Thunderbird version
    * Lightning version
    * Active Thunderbird add-ons
    * Log output

**Comment by Trim** [comment]
Your suggestion have been applied, thanks.

**Comment by advancingu** [comment]
Thanks!

**Pull request merged**